### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by robphoenix**
[skip-native]

### CODE REVIEW

File: packages/native-ui/src/hooks/useMedia.ts [hunk 1]
```diff
@@ -10,14 +10,15 @@ const useMedia = (): Partial<Record<keyof typeof breakpoints, boolean>> => {
...
-  }
+  // function isBreakpointKey(
+  //   value: keyof UnistylesBreakpoints,
+  //   breakpointsObj: typeof breakpoints
+  // ): value is BreakPointKeys {
+  //   return value in breakpointsObj;
+  // }
 
-  if (isBreakpointKey(breakpoint, breakpoints)) {
+  // if (isBreakpointKey(breakpoint, breakpoints)) {
```
Comment (robphoenix) [inline packages/native-ui/src/hooks/useMedia.ts:20]: @jordmccord I had to do this to make the linter happy after upgrading the deps. How would you like to handle this?

File: packages/native-ui/src/components/List/ListHeading/ListHeading.props.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import type { ViewProps } from 'react-native';
 
-interface ListHeadingBaseProps extends Omit<ViewProps, 'children'> {}
+type ListHeadingBaseProps = Omit<ViewProps, 'children'>;
```
Comment (jordmccord) [inline packages/native-ui/src/components/List/ListHeading/ListHeading.props.ts:3]: Triggers me mixing types and interfaces like this 😅

File: packages/native-ui/src/hooks/useMedia.ts (hunk 1 above)
Comment (jordmccord) [reply packages/native-ui/src/hooks/useMedia.ts:20]: What is the lint issue?

**Review by jordmccord — COMMENTED** [review]
Nice housekeeping 🧹 I'll try running everything,  I'm sure there was some issues when I tried to upgrade to pnpm 9 a while back

File: packages/native-ui/src/components/List/ListHeading/ListHeading.props.ts (hunk 2 above)
Comment (robphoenix) [reply packages/native-ui/src/components/List/ListHeading/ListHeading.props.ts:3]: aye. Just doing what the linter says 😆 we can always look at bypassing the rule?

File: packages/native-ui/src/hooks/useMedia.ts (hunk 1 above)
Comment (jordmccord) [reply packages/native-ui/src/hooks/useMedia.ts:20]: Ok got it 👍

**Review by jordmccord — APPROVED** [review]

**Pull request merged**